universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -99,11 +100,20 @@ public QuestionDefinition getQuestionDefinition() {
   }
 
   public LifecycleStage getLifecycleStage() {
-    return this.lifecycleStage;
+    return this.getQuestionDefinition().getLifecycleStage();
   }
 
   public void setLifecycleStage(LifecycleStage lifecycleStage) {
-    this.lifecycleStage = lifecycleStage;
+    try {
```

Comment:
Why is `lifecycleStage` only being set in the `QuestionDefinition` and also the `LifecycleStage lifecycleStage` field?

Reply:
Done.